templates/parts/credit.php

```diff
@@ -26,7 +35,11 @@
  */
 if ( (bool) apply_filters( 'primer_author_credit', true ) ) {
 
-	echo ' &mdash; ';
+	if ( ! empty( $copyright_text ) ) {
```

Comment:
@EvanHerman No need to use `empty` here since the variable is defined: `if ( $copyright_text ) {`